src/main/groovy/com/github/benmanes/gradle/versions/updates/Coordinate.groovy

```diff
@@ -54,24 +57,41 @@ class Coordinate implements Comparable<Coordinate> {
     return (result == 0) ? version.compareTo(coordinate.version) : result
   }
 
+  static Coordinate from(ExternalModuleDependency dependency) {
```

Comment:
This method was needed because ExternalModuleDependency implements both `ModuleVersionSelector` and `Dependency`, so it was going to the wrong method unless a more specific one was available.